TIKSN.Core/Web/Rest/IRestRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using TIKSN.Data;
+
+namespace TIKSN.Web.Rest
+{
+	public interface IRestRepository<TEntity, TIdentity> : IRepository<TEntity> where TEntity : IEntity<TIdentity> where TIdentity : IEquatable<TIdentity>
```

Comment:
Separate IRestRepository from IRepository, Also create IBalkRestRepository which will have AddRange and UpdateRange methods